schedule-tracking/src/main/java/org/motechproject/scheduletracking/domain/EnrollmentBuilder.java

```diff
@@ -7,74 +7,148 @@
 import java.util.Map;
 
 /**
- * The <code>EnrollmentBuilder</code> is a convenient class, that allows building {@link Enrollment} instances.
- * Once all required fields are set, call <code>toEnrollment()</code> method to get an instance of
- * Enrollment with the properties you have set.
+ * The <code>EnrollmentBuilder</code> is a convenient class, that allows building {@link org.motechproject.scheduletracking.domain.Enrollment}
+ * instances. Once all required fields are set, call <code>toEnrollment()</code> method to get an instance of
+ * enrollment with the properties you have set.
  */
 public class EnrollmentBuilder {
 
     private Enrollment enrollment;
 
+    /**
+     * Creates an EnrollmentBuilder.
+     */
     public EnrollmentBuilder() {
         enrollment = new Enrollment();
     }
 
+    /**
+     * Adds the metadata for the built enrollment.
```

Comment:
built suggests it has already been built. Perhaps change this to Adds the (something) to the enrollment in construction.
This applies to this whole builder.